src/uwtools/logger.py

```diff
@@ -105,14 +106,14 @@ def add_handlers(cls, logger: logging.Logger, handlers: List[logging.Handler]):
         -------
         logger
         """
-        for hh in handlers:
-            logger.addHandler(hh)
+        for h_h in handlers:
+            logger.addHandler(h_h)
```

Comment:
This choice isn't particularly more useful than the original. A lot of times I prefer the use of a singular/plural relationship between the items and the container of items. I'd suggest:

```
for handler in handlers:
    ...
```